pkg/store/tsdb_test.go

```diff
@@ -361,6 +365,22 @@ func TestTSDBStore_LabelValues(t *testing.T) {
 				return timestamp.FromTime(maxTime)
 			},
 		},
+		{
+			title:          "add another label value",
```

Comment:
Let's rename the title. If we only want to test the effects of matchers, actually we don't need to add another label value. We can just use `{Type: storepb.LabelMatcher_EQ, Name: "foo", Value: "test1"}` matcher and the expected value is `test1`.

What about renaming it to "check label value matcher".